Documentation/cheatsheet.rst

```diff
@@ -0,0 +1,137 @@
+#################
+Cilium Cheatsheet
+#################
+
+Below is a short list of examples of the various commands Cilium has to offer.
+If a command you use is missing please don’t hesitate to add it to one of the
+groups or create a new one.
+
+Basics
+======
+
+::
+
+        # Check the status of the agent
+
+        cilium status
+
+::
+
+        # Get the current agent configuration
+
+        cilium config
+
+Policy management
+=================
+
+::
+
+	# Importing a policy
+
+	cilium policy import <my-policy.json>
+
+::
+
+	# Get list of all imported policy rules
+
+	cilium policy get
+
+::
+
+	# Remove all policy
+
+	cilium policy delete --all
+
+Monitoring
+==========
+
+::
+
+	# Monitor cilium datapath notifications
+
+	cilium monitor 
+
+::
+
+	# Verbose output
+
+	cilium monitor -v
+
+::
+
+        # Show notifications only for dropped packet events
+
+        cilium monitor --related-to=<id>
+
+::
+
+	# Filter only the dropped packets
+
+	cilium monitor --type drop
+
+::
+
+	# Don't dissect packet payload, display payload in hex format
+
+	cilium monitor -v --hex
+
+Endpoints
+=========
+
+::
+
+	# Get list of all local endpoints
+
+	cilium endpoint list
+
+::
+
+        # Get detailed view of endpoint properties and state
+
+        cilium endpoint get <id>
+
+::
+
+        # Show recent endpoint specific log entries
```

Comment:
It turns out we store the endpoint logs in the BPF header with the other info and we also restore it, so it isn't just the recent log entries. I'm not sure if it's worth making that distinction here, however,

Reply:
Not following. How are the BPF headers which just include macros related to this?

Reply:
Oh, I meant it's not just the recent ones. It will be the last 256 logs, even if they are old. The current description is probably fine and more details might be confusing anyway.